packages/core/src/redocly/registry-api.ts

```diff
@@ -4,6 +4,8 @@ import { DEFAULT_REGION, DOMAINS } from '../config/config';
 import { isNotEmptyObject } from '../utils';
 const version = require('../../package.json').version;
 
+export const currentCommand = typeof process !== 'undefined' ? process.env?.REDOCLY_CLI_COMMAND || '' : '';
```

Comment:
Would this work?

```suggestion
export const currentCommand = process?.env?.REDOCLY_CLI_COMMAND || '';
```

Reply:
no, it will crash in the browser:

> ReferenceError: process is not defined

You can try it in your browser console.

Reply:
Ah, indeed.